### PR TITLE
fix(graphql): use output type definition to generate searchable filter

### PR DESCRIPTION
**Pull request by sundersc**
#### Description of changes
Use output type definition to generate searchable filters so that it includes the implicitly generated fields.

#### Description of how you validated changes
- yarn test

#### Checklist
- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by lazpavel — APPROVED** [review]
LGTM

File: packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts [hunk 1]
```diff
@@ -214,6 +216,10 @@ export class SearchableModelTransformer extends TransformerPluginBase {
   };
 
   transformSchema = (ctx: TransformerTransformSchemaStepContextProvider) => {
+    for(let name of this.searchableObjectNames) {
```
Comment (lazpavel) [inline packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts:219]: nit: use `const name`

**Review by lazpavel — APPROVED** [review]

**Review by cjihrig — APPROVED** [review]
LGTM

File: packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (sundersc) [reply packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts:219]: My bad. For some reason, `yarn lint` never pick up these things for me.

[the remark this answers is not in the document]
Comment (sundersc) [reply packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts]: Changed to `const` and added the missing whitespace.

Comment (sundersc) [reply packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts]: Updated.

**Pull request merged**